crates/e2e/tests/e2e/limit_orders.rs

```diff
@@ -73,6 +80,67 @@ async fn forked_node_gnosis_single_limit_order() {
     .await;
 }
 
+#[tokio::test]
```

Comment:
I think this was probably useful while debugging the code to see that you can pick up actually deployed pools but IMO this test should not be checked in since we can test the same thing with better control using a local test net.